books/free-programming-books-langs.md

```diff
@@ -445,7 +445,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Learning GNU C](https://download-mirror.savannah.gnu.org/releases/c-prog-book/learning_gnu_c.pdf) - Ciaran O’Riordan (PDF)
 * [Let us C](https://web.archive.org/web/20211006163041/http://pdvpmtasgaon.edu.in/uploads/dptcomputer/Let%20us%20c%20-%20yashwantkanetkar.pdf) - Yashavant Kanetkar (PDF) *(:card_file_box: archived)*
 * [Modeling with Data](https://ben.klemens.org/pdfs/gsl_stats.pdf) - Ben Klemens (PDF)
-* [Modern C](https://modernc.gforge.inria.fr) - Jens Gustedt (PDF)
+* [Modern C](https://hal.inria.fr/hal-02383654/document) - Jens Gustedt (PDF)
```

Comment:
Should the http version the right link? It works

http://modernc.gforge.inria.fr/

There are other downloadable content like examples there! Of course the PDF where https://hal.inria.fr/hal-02383654/document points to

Reply:
The link didn't work at my first visit. So I've found its PDF version and update it. But, it works now.